tests/cupy_tests/core_tests/test_ndarray_reduction.py

```diff
@@ -218,13 +219,20 @@ def test_ptp_nan_imag(self, xp, dtype):
     'order': ('C', 'F'),
 }))
 @testing.gpu
-@unittest.skipIf(cupy.cuda.cub_enabled is False, 'The CUB module is not built')
-class TestCUBreduction(unittest.TestCase):
+@unittest.skipUnless(cupy.cuda.cub_enabled, 'The CUB routine is not enabled')
+class TestCubReduction(unittest.TestCase):
+
+    def setUp(self):
+        self.old_accelerators = _accelerator.get_reduction_accelerators()
+        _accelerator.set_reduction_accelerators(['cub'])
+
+    def tearDown(self):
+        _accelerator.set_reduction_accelerators(self.old_accelerators)
```

Comment:
I think we should set routine accelerators here. It's used in `cupy/core/_routines_statistics.pyx` for min/max.

Reply:
Fixed!